src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -775,7 +782,7 @@ namespace rascal {
         auto smearing{downcast_atomic_smearing<AST>(this->atomic_smearing)};
 
         // a = 1 / (2*\sigma^2)
-        double fac_a{0.5 * pow(smearing->get_gaussian_sigma(center), -2)};
+        double fac_a{0.5 / pow(smearing->get_gaussian_sigma(center), 2_size_t)};
```

Comment:
why was this changed? To ensure `math::pow` over `std::pow`?

Reply:
yes it is

Reply:
Should we rename `math::pow` to `math::fast_pow` to be sure what we are using at a given point in code?

Reply:
seems too verbose for no reason to me. Function overloading does a good job here.

Reply:
One reason would be knowing immediately which function is called when reading the code

Reply:
this change goes beyond this PR so let's discuss this elsewhere. I personally prefer function overloading.

Reply:
#227